docs/operations.rst

```diff
@@ -114,8 +114,8 @@ This is typically caused by the query being configured to process only newly arr
 ------------------------------------------------------------
 Can’t create a stream from the output of windowed aggregate?
 ------------------------------------------------------------
-The output of a windowed aggregate is a record per grouping key and per window, and is not a single record. This is not
-currently supported in KSQL.
+The output of a windowed aggregate isn't a single record, so you can't create
```

Comment:
What does the output not being a single record have to do with not being able to define a stream? The output is a bunch of records, and whether I can define a stream on the records or not shouldn't depend on how the records relate to each other.

I thought the limitation was because KSQL doesn't support structured keys?

Reply:
I was trying to adapt the earlier, confusing comment, but I like your explanation much better!